mne/io/base.py

```diff
@@ -1268,6 +1268,26 @@ def crop(self, tmin=0.0, tmax=None, copy=None):
         return raw
 
     @verbose
+    def segment(self, segment_length=1., verbose=None, **kwargs):
```

Comment:
Segmentation is also used for artifact rejection in continuous data by ica and linear_regression_raw.